dotnet/src/SemanticKernel.Abstractions/Contents/ChatMessageContent.cs

```diff
@@ -15,6 +15,13 @@ namespace Microsoft.SemanticKernel;
 /// </summary>
 public class ChatMessageContent : KernelContent
 {
+    /// <summary>
+    /// Name of the author of the message
+    /// </summary>
+    [Experimental("SKEXP0001")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? Name { get; set; }
```

Comment:
Is "Name" the right name here? At first glance it would seem to be the name of the message rather than the name of the author of the message. Should it be "AuthorName" to correspond with "AuthorRole"?

Reply:
In the case of `Role` the type is `AuthorRole` and the property is `Role`.

I think the naming common usage and content and is consistent with the current type naming:  `Role`, `Content`, `Name` (sans prefixing).

I do understand and agree that overly simple naming can be problematic.  This often applies to types, such as `Node`, `Item`, `Kernel`, etc...  Because properties are type qualified, the concerns aren't identical (symbol collision).

Reply:
> In the case of Role the type is AuthorRole and the property is Role

It's unfortunate the author role property is just "Role", but even the it's arguable that's the role the message plays. It's a system message, a user message, etc, it's a property of the message.

This Name property really sounds to my ears like it's the name of the message. If there were an Id property, I'd again assume it was the id of the message.

> I think the naming common usage and content and is consistent with the current type naming: Role, Content, Name

Which highlights the problem. It's the content of the message but the name _of the author_ of the message.

Reply:
If this must deviate from the underlying model / spec...perhaps just `Author`.  

I'd have to add json mapping and reset functional testing.

Reply:
Consensus: `AuthorName` it is.  I'll ping when ready.